python/ray/train/lightning/lightning_trainer.py

```diff
@@ -237,6 +238,13 @@ class LightningTrainer(TorchTrainer):
     using the arguments provided in ``LightningConfigBuilder.fit_params()`` and then
     run ``pytorch_lightning.Trainer.fit``.
 
+    For more flexible and fine-grained control over `LightningModule` and `pl.Trainer`,
+    you can specify one or more customized PyTorch Lightning Callbacks
+    in `LightningConfigBuilder.trainer(callbacks=)`.
+
+    LightningTrainer requires ``pytorch_lightning>=1.6.5`` package, and it is tested
+    with ``pytorch_lightning==1.6.5`` and ``pytorch_lightning==2.0.0``.
```

Comment:
Should these be captured by an import error? When importing ptl, check the version and raise an error if we try to use LightningTrainer with an incompatible version.

Reply:
Some users complained that they don't know where to find the information about compatible ptl versions. So I put it here. What about we also add a import error to check the version compatibility.